js_modules/dagster-ui/packages/ui-core/src/instance/InstanceConcurrency.tsx

```diff
@@ -238,6 +238,10 @@ export const ConcurrencyLimits = ({
     limits.map(({concurrencyKey, slotCount}) => [concurrencyKey, slotCount]),
   );
 
+  const sortedLimits = React.useMemo(() => {
+    return [...limits].sort((a, b) => a.concurrencyKey.localeCompare(b.concurrencyKey));
```

Comment:
We have a collator that you can use here, which should also take care of any case sensitivity issues.

```
return [...limits].sort((a, b) => COMMON_COLLATOR.compare(a.concurrencyKey, b.concurrencyKey));
```